### ISSUE TITLE
你好，总共需要train多少个Epoch呀？

### ISSUE THREAD

**Issue by godad** [jinfagang/tensorflow_poems#10]
我用阿里云1核2G内存的ECS，跑了一晚上，还在跑。。。
总共需要train多少个Epoch呢？

[root@ali-bj ~]# screen -r 18900

[INFO] Epoch: 4 , batch: 537 , training loss: 3.252885
[INFO] Epoch: 4 , batch: 538 , training loss: 3.276884
[INFO] Epoch: 4 , batch: 539 , training loss: 3.551936
[INFO] Epoch: 4 , batch: 540 , training loss: 3.833653
[INFO] Epoch: 5 , batch: 0 , training loss: 3.327640
[INFO] Epoch: 5 , batch: 1 , training loss: 3.702888
[INFO] Epoch: 5 , batch: 2 , training loss: 3.101418
[INFO] Epoch: 5 , batch: 3 , training loss: 3.252736
[INFO] Epoch: 5 , batch: 4 , training loss: 3.087330
[INFO] Epoch: 5 , batch: 5 , training loss: 3.152049
[INFO] Epoch: 5 , batch: 6 , training loss: 4.089981
[INFO] Epoch: 5 , batch: 7 , training loss: 3.052967
[INFO] Epoch: 5 , batch: 8 , training loss: 2.722248
[INFO] Epoch: 5 , batch: 9 , training loss: 3.659889
[INFO] Epoch: 5 , batch: 10 , training loss: 2.923251
[INFO] Epoch: 5 , batch: 11 , training loss: 2.975899
[INFO] Epoch: 5 , batch: 12 , training loss: 2.727261
[INFO] Epoch: 5 , batch: 13 , training loss: 3.215596

**Comment by jinfagang** [comment]
4-5 enough, result would be fair,

**Comment by godad** [comment]
Thanks a lot. But how can I break it? ctrl+c doesn't seem to work...

**Comment by jinfagang** [comment]
<kbd>shift</kbd> + <kbd>ctrl</kdb> + <kbd>c</kdb> or.

**Comment by 625673575** [comment]
你用cpu跑的吧 我1070跑50 个echo大概两个小时

**Comment by mingjun4503** [comment]
想要个好结果，该怎么改epoch和batch_size?分别增大到多少合适？我用GPU跑
